docs/requirements.txt

```diff
@@ -0,0 +1 @@
+numpydoc
```

Comment:
Is `pygments` present from some other dependency chain? I've a recollection it's needed but guess it's coming in from somewhere else as the demo URL renders fine.

Reply:
Looks like RTD downloads Sphinx by default, and `pygments` are a dependency there. Ends up working okay.

Reply:
Then I guess leave as-is until proven otherwise :)